provisioners/src/main/java/org/jboss/intersmash/provision/openshift/OpenShiftAIOpenShiftOperatorProvisioner.java

```diff
@@ -47,7 +49,7 @@ public class OpenShiftAIOpenShiftOperatorProvisioner
 		// ... and common OpenShift provisioning logic, too
 		implements OpenShiftProvisioner<OpenShiftAIOperatorApplication> {
 
-	private static final String OPENSHIFT_OPERATORS_NAMESPACE = "openshift-operators";
+	private static final String OPENSHIFT_OPERATORS_NAMESPACE = "openshift-operators"; // --> redhat-ods-operator
```

Comment:
this one could be a TODO or a separate issue about using the namespace mentioned in https://ai-on-openshift.io/odh-rhoai/gitops/#operator-installation